cp/cp_config.c

```diff
@@ -220,6 +220,32 @@ init_spgwc_dynamic_config(struct app_config *cfg )
 		} else {
 			RTE_LOG_DP(ERR, CP, "TAC not found in the configuration file\n");
 		}
+		entry=rte_cfgfile_get_entry(file, sectionname, "NUM_IMSIs");
+        if (entry)
+        {            
+            dpInfo->key.num_imsis = atoi(entry);
+            entry = rte_cfgfile_get_entry(file, sectionname, "IMSIs");
+            if(entry)
+            {   
+                char * imsis = "";             
+                memcpy(&imsis, &entry, sizeof(entry));
```

Comment:
memory allocation is not required to copy the entry here ?

Reply:
it is not required, but I will allocate memory for it, so I can raise error if there is no sufficient memory